src/states/OptionsMenu.tscn

```diff
@@ -34,7 +34,7 @@ align = 1
 [node name="G" type="GridContainer" parent="C/V"]
 margin_top = 86.0
 margin_right = 389.0
-margin_bottom = 151.0
+margin_bottom = 180.0
```

Comment:
Im trusting you that all of these new margins look good.

Reply:
I just added more shit to the scene, godot picked those margins for me since it's in a center container and it shifted things around